docs/governance/community-guidelines/releases/cmb.md

```diff
@@ -0,0 +1,106 @@
+# Change Management Board
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines and defines the guidelines for the Change Management Board (CMB) for the Common Cloud Controls (CCC) project.
+
+The CMB is a body of representatives from financial institutions of varying sizes and types. Its primary role is to review and approve changes and new catalogs that are within the Release Candidate. The CMB collectively represents end-user stakeholders, ensuring that each artifact is adaptable to the needs of a wide range of institutions while maintaining consistency and integrity across the board.
```

Comment:
Who decides who's on the CMB? And how big is it? Does it need to be unanimous approval? Or a simple majority? Is there a constitution or other definition document for it?

Reply:
No max, the idea is to give the community an opportunity to review and approve— and give everyone confidence that at least a certain number of organizations have approved it as ready